integration-tests/known-contracts-ProxfordS.ts

```diff
@@ -0,0 +1,8 @@
+import { KnownContracts } from './known-contracts';
```

Comment:
Is the renaming of this file intentional?

Reply:
yes it's substring from the protocol hash which is supposedly auto-generated from `integration-tests/originate-known-contracts.ts`